manual_tests/lib/gh/check.py

```diff
@@ -49,7 +51,7 @@ def get_runs(self) -> Generator[CheckRun]:
 
         for obj in get_runs_json(self.pr.url):
             run = CheckRun.from_json(obj)
-            if run.name == self.name:
+            if run.workflowName == self.workflow_name and run.name == self.name:
```

Comment:
I don't see that you added a `workflowName` property to CheckRun.

Reply:
It is already there